.github/workflows/contract.yml

```diff
@@ -30,7 +30,7 @@ jobs:
 
       - uses: actions/setup-go@v2
         with:
-          go-version: '1.16.7'
+          go-version: '1.17.1'
```

Comment:
1.17.2 is out, we should use it.

Question though: is there a way to instruct these actions to use "1.17.latest-patch"? If so, WDYT of using it instead of pinning a patch version of Go? There are trade-offs here of course, but go patch versions (and minor versions for that matter!) are very good about compatibility, and IMO the benefits to not pinning a patch outweigh the unpredictability downside. Up to you though!

This note applies to similar gh-actions manifests too of course